scripts/deploy.py

```diff
@@ -24,12 +26,14 @@ def get_address(msg: str) -> str:
 
 def main():
     print(f"You are using the '{network.show_active()}' network")
-    dev = accounts.load("dev")
+    dev = accounts.load(click.prompt("Account", type=click.Choice(accounts.load())))
     print(f"You are using: 'dev' [{dev.address}]")
     token = Token.at(get_address("ERC20 Token: "))
-    gov = get_address("Yearn Governance: ")
-    rewards = get_address("Rewards contract: ")
-    name = input(f"Set description ['yearn {token.name()}']: ") or ""
+    gov = get_address("Yearn Governance (0xFEB4acf3df3cDEA7399794D0869ef76A6EfAff52): ")
+    rewards = get_address(
+        "Rewards contract (0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde): "
+    )
+    name = input(f"Set description ['{token.name()} vault']: ") or ""
```

Comment:
This is technically correct, but the update was never done
```suggestion
    name = input(f"Set description ['{token.name()} yVault']: ") or ""
```